### ISSUE TITLE
Format in validation length will cause error

### ISSUE THREAD

**Issue by marcinma** [mission-liao/pyswagger#33]
In primitives lines 116 - 119

```python
if obj.minItems and len(self) < obj.minItems:
            raise errs.ValidationError('Array should be more than {0}, not {1}'.format(o.minItems, len(self)))
if obj.maxItems and len(self) > obj.maxItems:
            raise errs.ValidationError('Array should be less than {0}, not {1}'.format(o.maxItems, len(self)))
```

o is not defined and should be replaced to obj

**Comment by mission-liao** [comment]
thanks for pointing out, there must be some test case missed.

**Comment by mission-liao** [comment]
thx for reporting issues, this issue is fixed in v0.8.9.
